linux/requirements.bsh

```diff
@@ -91,8 +91,13 @@ function _meet_requirements()
 
   while (( $# )); do
     case "${1}" in
+      ==*)
```

Comment:
Can this be combined with =*?

`==*|=*)`

Reply:
Oh, nevermind, they are different

Reply:
If I combine the two, I need a way to appropriately get `${1:2}` for `==*` and `${1:1}` for `=*`.  You know how to do that?